src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2475,14 +2471,14 @@ When nodes are removed from the slave cluster, or if nodes go down or come back
 
 ==== Keeping Track of Logs
 
-Each master cluster region server has its own znode in the replication znodes hierarchy.
-It contains one znode per peer cluster (if 5 slave clusters, 5 znodes are created), and each of these contain a queue of WALs to process.
+Each master cluster region server has its queue state in the hbase:replication table.
+It contains one row per peer cluster (if 5 slave clusters, 5 rows are created), and each of these contain a queue of WALs to process.
```

Comment:
Here things are a bit different. For zookeeper, it is like a tree, we have a znode for a peer cluster, but under the znode we have lots of files.
But for table based implementation, we have server name in row key, which means we will have lots of rows for a given peer...